### PR TITLE
bug(GRW-952): details modal save

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## What?

Fix a bug where Details Modal wasn't triggering a API request when "Save" button is clicked;

**Ticket(s): [GRW-952](https://hedvig.atlassian.net/browse/GRW-952)**

## Screenshots / recordings 

https://user-images.githubusercontent.com/19200662/160577492-85ea90d2-ee20-47c8-a756-ab5322341270.mov

### CODE REVIEW

File: src/client/components/ModalNew.tsx [hunk 1]
```diff
@@ -171,7 +170,7 @@ const Footer = styled.div<ModalFooterProps>`
   align-items: center;
   justify-content: center;
   ${({ sticky }) =>
-    !sticky &&
+    sticky &&
```
Comment (robinandeer) [inline src/client/components/ModalNew.tsx:173]: haha just wondering why this changes?

**Review by robinandeer — APPROVED** [review]
Makes sense to me -- a little strange that <Formik> doesn't render a <form> element on it's own tho 😅

File: src/client/components/ModalNew.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply src/client/components/ModalNew.tsx:173]: It's just that before `position: sticky` (and "associate" styles) were being applied when `sticky` prop was `false` which is misleading is my opinion: you want to add the styles when "you ask" for them - `sticky` being `true`.

It was working before because `sticky` prop wasn't being provided to <Footer /> component: `!undefined && ...`. As you can see [here](https://github.com/HedvigInsurance/web-onboarding/pull/890/files#diff-e9fa83e568fe0a1210f47c062d4d43bb35e20fdcef8ac7f7d0ef6978fa685207R183)

Comment (simonauner) [reply src/client/components/ModalNew.tsx:173]: I forgot to pass `sticky` to the styled element, so `sticky` was `falsy`. Now that @guilhermespopolin forwards `sticky` the condition must be switched
